tue-fancydesktop/install.bash

```diff
@@ -1,11 +1,12 @@
-#!/bin/bash
+#! /usr/bin/env bash
+
 source /etc/lsb-release
 
 # copying shortcuts to desktop
-cp $TUE_DIR/installer/targets/tue-fancydesktop/shortcuts/.*.desktop ~/Desktop
+tue-install-cp shortcuts/.*.desktop ~/Desktop
```

Comment:
Does tue-install-cp work with patterns?

Reply:
Good point. Need to check that.

Reply:
At the moment it doesn't work but I have found the solution for which I'll open a PR now.

Reply:
I have tested it, it works in the installer,  but if load the functions and run it in the terminal yourself, it fails.